pkg/cmd/root/root_test.go

```diff
@@ -26,3 +32,81 @@ func TestRootSetsValidLogLevel(t *testing.T) {
 	ConfigureGlobalLogger("panic")
 	assert.Equal(t, log.PanicLevel, log.DefaultLogger.Level)
 }
+
+func TestCheckDuplicateCmds(t *testing.T) {
+	log.DefaultLogger.SetLevel(log.Level(8))
```

Comment:
Would a setup tear down be appropriate here? You are modifying the default logger, may want to set it back after you have touched it.

Reply:
https://pkg.go.dev/testing#T.Cleanup